src/soundfile.js

```diff
@@ -1259,6 +1261,17 @@ define(function (require) {
       }
     };
 
+    // if the AudioWorkletNode is actually a ScriptProcessorNode created via polyfill,
+    // make sure that our chosen buffer size isn't smaller than the buffer size automatically
+    // selected by the polyfill
+    // reference: https://github.com/GoogleChromeLabs/audioworklet-polyfill/issues/13#issuecomment-425014930
+    if (self._workletNode instanceof ScriptProcessorNode) {
+      self._workletNode.port.postMessage({
+        name: 'bufferSize',
+        bufferSize: Math.max(workletBufferSize, self._workletNode.bufferSize)
```

Comment:
is it better to pick the max, or—in cases where AudioWorklet is not supported—to _always_ go with the script processor node buffer size?

Reply:
yeah, I agree that it's probably better to always go with the script processor node buffer size - I've used that approach in my latest commit